src/frontend/src/common/queries/auth.ts

```diff
@@ -36,17 +31,17 @@ const mapUserData = (obj: any): User => {
 };
 
 function mapGroupData(obj: any): Group {
-  return <Group>{
+  return {
     id: obj.id,
     name: obj.name,
   };
 }
 
-export const fetchUserInfo = (): Promise<User> => {
+export const fetchUserInfo = (): Promise<RawUserRequest> => {
```

Comment:
I see now that this type gets used both for the response from a GET and also the payload structure for a PUT. Maybe a better name would just be `RawUser` or something since it's used both in request and response